src/hera.cpp

```diff
@@ -191,6 +195,49 @@ vector<uint8_t> evm2wasm(evmc_context* context, vector<uint8_t> const& input) {
   return ret;
 }
 
+// Calls the runevm contract.
+// @returns a wasm-based evm interpreter.
+vector<uint8_t> runevm(evmc_context* context, WasmEngine& engine, vector<uint8_t> code) {
```

Comment:
Can you make a helper `locallyExecuteSystemContract` similar to `callSystemContract`?